### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by katestange** [numberscope/backscope#99]
@gwhitney wrote in #75: 

Most python projects have a "pyproject.toml" file (see https://packaging.python.org/en/latest/tutorials/packaging-projects/) that specifies a current version number of the project. That's updated (manually) when a new release of the project is made. But I see that backscope has no such thing.

We might want to implement such a thing, and then have an endpoint to display it.

**Comment by gwhitney** [comment]
Yes, we might. It's most important in the context of publishing a package on PyPI, which I don't particularly see for backscope, but explicit server versioning might still be useful for syncing between backscope and frontscope changes. Since it's unlikely we will act on this any time soon, we might want to move this issue to a Discussion.
